### ISSUE TITLE
Make Path annotation content configurable from properties

### ISSUE THREAD

**Issue by Saladoc** [quarkusio/quarkus#27746]
### Description

With Spring MVC, it is possible to use placeholders with Path configurations. In `@RequestMapping("${myapp.path}")` the placeholder will be replaced with the myapp.path property.

When looking for a similar feature in Quarkus (or really any feature to dynamically configure the path of a specific endpoint) I was informed that currently no such feature exists in quarkus.

### Implementation ideas

Honestly, I wouldn't know how to go about this. I realize that this might open a whole can of worms since if it would even remotely be like SpEL so I'm open to really any way that conveniently lets people configure a path from properties, environment variables or both, be it a quarkus feature or a neat workaround.

**Comment by Matthiasbrat** [comment]
Hi, I answered your question on stackoverflow. Please tell me if you think it solved your issue. Have a nice day
[https://stackoverflow.com/questions/73614638/quarkus-using-placeholders-in-path#answer-74877497](link to the stackoverflow question)

**Comment by Hubbitus** [comment]
Hello.
Sorry, is there some solution or at least workaround?

**Comment by FroMage** [comment]
You can set a path prefix (https://quarkus.io/guides/http-reference#context-path) via configuration, but that's for every endpoint. There's no workaround, no, sorry.

**Comment by ghost** [comment]
after 1 year is there any solution for this

**Comment by Matthiasbrat** [comment]
> after 1 year is there any solution for this

Nope, I had to make another solution. I'm sure you'll find a way like others 👍

**Comment by ghost** [comment]
Ooh Thanks i had to fix , got a solution
